modules/objc/generator/gen_objc.py

```diff
@@ -1,23 +1,20 @@
 #!/usr/bin/env python
 
+from __future__ import print_function, unicode_literals
```

Comment:
Probably `from __future__ import unicode_literals` is not used too (non-ASCII symbols should be avoided in .py source code).

Reply:
Turns out that the `unicode_literals` import is needed to prevent this failure on Python 2:

```
  File "/Volumes/build-storage/build/precommit_ios/opencv/modules/objc/generator/../generator/gen_objc.py", line 1224, in gen_class
    {0}\n}};\n\n""".format(",\n    ".join(["%s = %s" % (c.name, c.value) for c in consts]), typeName, typeName)
TypeError: unicode argument expected, got 'str'
```